src/test/java/com/iexec/sms/tee/config/TeeWorkerInternalConfigurationTests.java

```diff
@@ -29,52 +33,67 @@ class TeeWorkerInternalConfigurationTests {
     private static final String FINGERPRINT = "fingerprint";
     private static final String ENTRYPOINT = "entrypoint";
     private static final String LAS_IMAGE = "lasImage";
-    private static final long HEAP_SIZE_GB = 3;
+    private static final DataSize HEAP_SIZE = DataSize.parse("3GB");
     private static final long HEAP_SIZE_B = 3221225472L;
 
-    private final TeeAppProperties preComputeProperties = TeeAppProperties.builder()
-            .image("preComputeImage")
-            .fingerprint("preComputeFingerprint")
-            .entrypoint("preComputeEntrypoint")
-            .heapSizeInBytes(1L)
-            .build();
-    private final TeeAppProperties postComputeProperties = TeeAppProperties.builder()
-            .image("postComputeImage")
-            .fingerprint("postComputeFingerprint")
-            .entrypoint("postComputeEntrypoint")
-            .heapSizeInBytes(1L)
-            .build();
+    private TeeWorkerInternalConfiguration teeWorkerInternalConfiguration;
+    private TeeWorkerPipelineConfiguration pipelineConfig;
+    private TeeWorkerPipelineConfiguration.StageConfig validStageConfig;
 
+    private TeeAppProperties preComputeProperties;
+    private TeeAppProperties postComputeProperties;
 
-    private final TeeWorkerInternalConfiguration teeWorkerInternalConfiguration =
-            new TeeWorkerInternalConfiguration();
+    @BeforeEach
+    void setUp() {
+        teeWorkerInternalConfiguration = new TeeWorkerInternalConfiguration();
 
-    // region preComputeProperties
-    @Test
-    void preComputeProperties() {
-        TeeAppProperties properties = teeWorkerInternalConfiguration.preComputeProperties(
+        validStageConfig = new TeeWorkerPipelineConfiguration.StageConfig(
                 IMAGE,
                 FINGERPRINT,
-                ENTRYPOINT,
-                HEAP_SIZE_GB
+                HEAP_SIZE,
+                ENTRYPOINT
+        );
+
+        final TeeWorkerPipelineConfiguration.Pipeline pipeline = new TeeWorkerPipelineConfiguration.Pipeline(
+                "v5",
+                validStageConfig,
+                validStageConfig
         );
 
+        pipelineConfig = new TeeWorkerPipelineConfiguration(
+                Collections.singletonList(pipeline)
+        );
+
+        preComputeProperties = TeeAppProperties.builder()
+                .image("preComputeImage")
+                .fingerprint("preComputeFingerprint")
+                .entrypoint("preComputeEntrypoint")
+                .heapSizeInBytes(1L)
+                .build();
+
+        postComputeProperties = TeeAppProperties.builder()
+                .image("postComputeImage")
+                .fingerprint("postComputeFingerprint")
+                .entrypoint("postComputeEntrypoint")
+                .heapSizeInBytes(1L)
+                .build();
```

Comment:
The `setUp` method is called before every `@Test` annotated method. Those objects are recreated identically before each test, which is not efficient. We can and should keep the way those fields were created.
In this specific situation, it creates a code change where it was not necessary. This creates a bigger PR without really adding new value.